test/cpp/jit/test_backends.cpp

```diff
@@ -0,0 +1,46 @@
+#include <test/cpp/jit/test_utils.h>
+
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/backends/backend.h>
+
+namespace torch {
+namespace jit {
+
+// This test JIT backend is intended to do the minimal amount of work
+// necessary to test that the JIT backend registration endpoints and
+// code generation are working correctly. It is not intended to
+// produce numerically correct results.
+RegisterBackend backend({
```

Comment:
This backend is maybe *too* simple. I'd be open to modifying it so that it uses a `GraphExecutor` instance to run a graph.